modules/sor/sorV2/lib/static.ts

```diff
@@ -37,6 +39,8 @@ export async function sorGetSwapsWithPools(
             case 'META_STABLE':
                 basePools.push(MetaStablePool.fromPrismaPool(prismaPool));
                 break;
+            case 'STABLE':
+                basePools.push(StablePool.fromPrismaPool(prismaPool));
```

Comment:
this is actually the legacy stable pools, do we need them here?

Reply:
I thought of using it for the new Stable types, but maybe it'll be better to create a new pool type for Stable V3 pools